graphql/directive_test.go

```diff
@@ -218,7 +234,87 @@ func setupPermissions(t *testing.T) {
 	}
 	require.NoError(t, roleManager.UpdateRole(logViewRole))
 }
-
+func TestRequireHostAccess(t *testing.T) {
```

Comment:
The references to `.Equal` or `.EqualError` in this file should use `assert` rather than `require` for codebase style consistency.